spec/index.bs

```diff
@@ -12129,30 +12129,40 @@ interface GPUCanvasContext {
                         this generates and error and returns an [=invalid=] {{GPUTexture}}.
                         Some validation here is redundant with that done in {{GPUCanvasContext/configure()}}.
                         Implementations **must not** skip this redundant validation.
+                    1. [=Queue a microtask=] to [$expire access to the current texture$].
                 1. Return |this|.{{GPUCanvasContext/[[currentTexture]]}}.
             </div>
         </div>
 
         Note: The same {{GPUTexture}} object will be returned by every
-        call to {{GPUCanvasContext/getCurrentTexture()}} made within the same frame (i.e. between
-        invocations of "[$update the rendering of the WebGPU canvas$]"), even if that {{GPUTexture}}
-        is destroyed, failed validation, or failed to allocate, **unless** the current texture has
-        been removed (in [$Replace the drawing buffer$]).
+        call to {{GPUCanvasContext/getCurrentTexture()}} made until the access to the texture
+        [$expire access to the current texture|expires$], even if that {{GPUTexture}} is destroyed,
+        failed validation, or failed to allocate, **unless** the current texture has been removed
+        (in [$Replace the drawing buffer$]).
 </dl>
 
 <div algorithm>
-    In the "update the rendering or user interface of that `Document`" sub-step of the
-    "[=Update the rendering=]" step of the HTML processing model, each {{GPUCanvasContext}} |context|
-    must <dfn abstract-op>update the rendering of the WebGPU canvas</dfn> by running the following steps:
+    To <dfn abstract-op>expire access to the current texture</dfn> run the following steps:
 
-    1. Update the rendering of |context|.{{GPUCanvasContext/canvas}} with
-        [$get a copy of the image contents of a context|a copy of the image contents$] of |context|.
     1. If |context|.{{GPUCanvasContext/[[currentTexture]]}} is not `null`:
         1. Call |context|.{{GPUCanvasContext/[[currentTexture]]}}.{{GPUTexture/destroy()}}
             (without destroying |context|.{{GPUCanvasContext/[[drawingBuffer]]}})
             to terminate write access to the image.
         1. Set |context|.{{GPUCanvasContext/[[currentTexture]]}} to `null`.
 
+    Note: When the current texture expires the page no longer has the ability to write to it, but
+    the contents remain accessible for [$update the rendering of the WebGPU canvas|compositing$] or
+    [$get a copy of the image contents of a context|copying$].
+</div>
+
+<div algorithm>
+    In the "update the rendering or user interface of that `Document`" sub-step of the
+    "[=Update the rendering=]" step of the HTML processing model, each {{GPUCanvasContext}} |context|
+    must <dfn abstract-op>update the rendering of the WebGPU canvas</dfn> by running the following steps:
+
```

Comment:
[=Assert=] that the current texture is destroyed?

Reply:
Good idea! Done.